packages/web-app-files/src/views/shares/SharedWithOthers.vue

```diff
@@ -102,18 +102,7 @@ export default defineComponent({
   setup() {
     const store = useStore()
     const getSpace = (resource: Resource): SpaceResource => {
-      const storageId = resource.storageId
-      // FIXME: Once we have the shareId in the OCS response, we can check for that and early return the share
-      const space = store.getters['runtime/spaces/spaces'].find((space) => space.id === storageId)
-      if (space) {
-        return space
-      }
-
-      return buildShareSpaceResource({
-        shareId: resource.shareId,
-        shareName: resource.name,
-        serverUrl: configurationManager.serverUrl
-      })
+      return getSpaceFromResource({ spaces: store.getters['runtime/spaces/spaces'], resource })
```

Comment:
Lambda without return possible ?

Reply:
Yeah, but honestly I'm not the biggest fan of using it for computed properties, because when I want to add code or console.log something, I always need to re-add the braces.